src/app/features/attendance/display-attendance/display-attendance.component.html

```
@@ -1,8 +1,10 @@
-@if (items().length && percentage() !== undefined) {
+@if (items().length) {
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)